src/dataCorrectionsTool/incentiveEligibilityTool.js

```diff
@@ -0,0 +1,416 @@
+import fieldMapping from '../fieldToConceptIdMapping.js';
+import { dashboardNavBarLinks, removeActiveClass } from '../navigationBar.js';
+import { renderParticipantHeader } from '../participantHeader.js';
+import { handleBackToToolSelect, displayDataCorrectionsNavbar, setActiveDataCorrectionsTab } from './dataCorrectionsHelpers.js';
+import { showAnimation, hideAnimation, baseAPI, getIdToken, getDataAttributes, triggerNotificationBanner } from '../utils.js';
+
+let participantPaymentRound = null;
+let isEligibleForIncentiveUpdate = null;
+
+const conceptIdToPaymentRoundMapping = {
+    266600170: 'baseline',
+}
+
+export const setupIncentiveEligibilityToolPage = (participant) => { 
+    if (participant !== undefined) {
+        const isParent = localStorage.getItem; 
```

Comment:
This should probably actually be getting something from localStorage instead of passing in the actual localStorage.getItem function, no?

Reply:
Good catch, that is correct. This will need `isParent` from localStorage.

`const isParent = localStorage.getItem('isParent');`